proposals/wac-ucr/index.bs

```diff
@@ -1111,16 +1532,259 @@ specially generated link to access the document with [=read access=]. He sets
+
+<div class="assertion">    
+  Related use cases: [[#inheritance-appendonly]]
+</div>
+    
+### The system shall allow for the access permissions associated with a certain resource to be read if the [=agent=] has inherited that permission from the [=collection=] the [=resource=] is a member of. ### {#req-inheritance-read-permissions}
+
+<div class="assertion">    
+  Related use cases: [[#inheritance-control]]
+</div>
+    
+### The system shall allow for the access permissions associated with a certain resource to be changed if the [=agent=] has inherited that permission from the [=collection=] the [=resource=] is a member of. ### {#req-inheritance-change-permissions}
+
+<div class="assertion">    
+  Related use cases: [[#inheritance-control]]
+</div>
+
+## Conditional access ## {#req-conditional}
+
+### The system shall limit the ability to access a certain [=resource=] by a given start and/or end data and time.### {#req-conditional-time}
+
+<div class="assertion">    
+  Related use cases: [[#conditional-time]]
+</div>
+
+### The system shall limit the ability to access a certain [=resource=] by a tag associated with that resource.### {#req-conditional-tag}
+
+<div class="assertion">    
+  Related use cases: [[#conditional-tag]]
+</div>
+    
+### The system shall limit the ability to access a certain [=resource=] based on the existence of a specific relationship with another [=resource=].### {#req-conditional-relationship}
+
+<div class="assertion">    
+  Related use cases: [[#conditional-relationship]]
+</div>
+    
+### The system shall limit access to only a subset of data in a certain [=resource=] based on supplied filter criteria.### {#req-conditional-filter}
+
+<div class="assertion">    
+  Related use cases: [[#conditional-filter]]
+</div>
+
+### The system shall allow the [=access modes=] and/or conditions of a given access permission for a certain [=resource=] or [=collection=] to change after other specified conditions have been satisfied.### {#req-conditional-action}
+
+<div class="assertion">    
+  Related use cases: [[#conditional-action]]
+</div>
+  
+### The system shall limit the ability to read, create, or change only those access permissions for a given [=resource=] or [=collection=] that apply to a specified group of [=agents=].### {#req-conditional-control}
+
+<div class="assertion">    
+  Related use cases: [[#conditional-control]]
+</div>
+
+### The system shall limit the ability to read, create, or change access permissions for [=resources=] associated with a particular tag.### {#req-conditional-control-tag}
+
+<div class="assertion">    
+  Related use cases: [[#conditional-control]]
+</div>
 
 Limitations of Legacy Web Access Control {#limitations}
 ================================================================================
```

Comment:
This section should perhaps be put in terms of requirements. But since it is put in terms of use cases: 
 * 2.6.3. Minimal Credential Disclosure [requirement: ability to read access control rules by client]
 * 2.6.4. Limit information disclosure through URI [I think, the current system forces decisions to be made on URL patterns to decide what is a container or a URI -- now 100% sure]
 * 2.4.6 Conditional access by payment

Reply:
Have added the suggested use cases in https://github.com/solid/authorization-panel/pull/152/commits/7086b9921b09ca04c5380a3dfb740403c3e0b667, except for the limit information disclosure through URI, which (per [this thread](https://github.com/solid/authorization-panel/pull/152#discussion_r572451608) may be treading into sticky ground as rightly brought up by @matthieubosquet)

Reply:
Resolved last item in https://github.com/solid/authorization-panel/pull/152/commits/83644fc68adde83270fcec85e6574974dffd0888